projects/extensions/data-grid/grid.component.ts

```diff
@@ -48,17 +47,19 @@ import { MtxGridService } from './grid.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
   animations: [
     trigger('expansion', [
-      state('collapsed', style({ height: '0', minHeight: '0', visibility: 'hidden' })),
-      state('expanded', style({ height: '*', visibility: 'visible' })),
+      state('collapsed', style({height: '0', minHeight: '0', visibility: 'hidden'})),
+      state('expanded', style({height: '*', visibility: 'visible'})),
       transition('expanded <=> collapsed', animate('225ms cubic-bezier(0.4, 0.0, 0.2, 1)')),
     ]),
   ],
 })
-export class MtxGridComponent implements OnInit, OnChanges, AfterViewInit, OnDestroy {
+export class MtxGridComponent implements OnChanges, AfterViewInit, OnDestroy {
   @ViewChild(MatPaginator) paginator!: MatPaginator;
   @ViewChild(MatSort) sort!: MatSort;
   @ViewChild('columnMenu') columnMenu!: MtxGridColumnMenu;
+  @ViewChild('table') table!: MatTable<any>;

```

Comment:
It's unnecessary to use a template variable (`#table`) to get the table reference.
```ts
@ViewChild(MatTable) table!: MatTable<any>;
```